docs/src/main/asciidoc/spring-cloud-wiremock.adoc

```diff
@@ -288,8 +288,9 @@ will generate a stub something like this:
 NOTE: You can use either the `wiremock()` method or the `jsonPath()`
 and `contentType()` methods to create request matchers, but not both.
 
-On the consumer side, assuming the `resource.json` generated above is
-available on the classpath, you can create a stub using WireMock in a
+On the consumer side, you can make the `resource.json` generated above
+available on the classpath (by http://cloud.spring.io/spring-cloud-contract/spring-cloud-contract.html#_publishing_stubs_as_jars[publishing stubs as JARs] for example).
```

Comment:
We're in `1.0.x` and pointing to docs from `master` . I think we should point to `1.0.x`.